<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockController</class>
 <widget class="QWidget" name="StockController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="inTab">
      <attribute name="title">
       <string>Entrant</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>5</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Date(Année)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="date_in_edit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>1000</number>
              </property>
              <property name="maximum">
               <number>4000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showInCheck">
              <property name="text">
               <string>Tout afficher</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="newBtn">
           <property name="text">
            <string>Nouvel enregistrement</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="inTable">
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Désignation</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Caractéristiques</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantité</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Prix unitaire</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Montant total</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="outTab">
      <attribute name="title">
       <string>Sortie</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Date(Année)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="date_out_edit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="minimum">
             <number>1000</number>
            </property>
            <property name="maximum">
             <number>4000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showOutCheck">
            <property name="text">
             <string>Tout afficher</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newBtn_2">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Nouvel Enregistrement</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="outTable">
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Désignation</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Caractéristiques</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantité</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="interTab">
      <attribute name="title">
       <string>Inventaire de stocks</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QComboBox" name="invCombo">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="showInvCheck">
           <property name="text">
            <string>Tout afficher</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="invtable">
         <column>
          <property name="text">
           <string>Désignation</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Caractéristiques</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantité entrants</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantité sortants</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Stock restant</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="inverTab">
      <attribute name="title">
       <string>Interchangeabilité</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
